src/ui/Button.tsx

```diff
@@ -70,6 +109,8 @@ export const Button = forwardRef<HTMLButtonElement, Props>(
         onClick={to ? () => navigate(to) : onClick}
       >
         {children}
+        {type === "secondary" && <IconDownload src={iconDownload} />}
+        {type === "tertiary" && <IconGo src={iconGo} />}
```

Comment:
Adding icon to the content of the button should not be based on the type of the button as this component supposed to be shared for not only current usecases but all possible future use cases. We don't know if all future secondary buttons will all get a download icon.